rules/filename-case.js

```diff
@@ -67,10 +90,25 @@ function splitFilename(filename) {
 	};
 }
 
+/**
+Turns `[a, b, c]` into `a, b or c`.
```

Comment:
```suggestion
Turns `[a, b, c]` into `a, b, or c`.
```

I prefer Oxford comma: https://www.grammarly.com/blog/what-is-the-oxford-comma-and-why-do-people-care-so-much-about-it/

Reply:
Never knew this is a thing ^^

Reply:
resolved